src/main/java/racing/controller/GameController.java

```diff
@@ -0,0 +1,22 @@
+package racing.controller;
+
+import racing.domain.RacingGame;
+import racing.view.output.OutputView;
+
+public class GameController {
+    private final RacingGame racingGame;
+
+    public GameController(RacingGame racingGame) {
+        this.racingGame = racingGame;
+    }
+
+    public void run() {
+        OutputView.printPhrase();
+        int tryCount = racingGame.getTryCount();
+        while (tryCount-- > 0) {
```

Comment:
```suggestion
        while (racingGame.getTryCount()-- > 0) {
```
한군데서 밖에 쓰이지 않다면, 이렇게 설정해줘도 좋을 것 같아요 :)